peachjam_search/templates/peachjam_search/_saved_search_modal.html

```diff
@@ -27,6 +27,20 @@ <h1 id="saved-search-modal-label" class="modal-title fs-5">{% trans 'Search aler
                         placeholder="{% trans "A note to yourself about this search" %}">{{ form.note.value|default_if_none:'' }}</textarea>
             </div>
           </form>
+        {% elif search_alert_limit_reached %}
+          <div class="alert alert-warning d-flex">
+            <div class="me-2">
+              <i class="bi bi-exclamation-triangle-fill"></i>
+            </div>
+            <div>
+              <div class="fw-bold">{% trans 'You have reached your limit for search alerts.' %}</div>
+              {% if search_alert_upgrade %}
+                {% blocktrans trimmed with product=search_alert_upgrade.name %}
+                  <div>To add more, please upgrade to {{ product }}.</div>
```

Comment:
Minor: it's better not to have HTML in blocktrans if it can be avoided, since it complicates translation slightly.